hack/deployer/runner/gke.go

```diff
@@ -79,7 +78,10 @@ func (gdf *GkeDriverFactory) Create(plan Plan) (Driver, error) {
 }
 
 func (d *GkeDriver) Execute() error {
-	if err := d.auth(); err != nil {
+	if err := authToGCP(
+		*d.plan.VaultInfo, GkeVaultPath, GkeServiceAccountVaultFieldName,
```

Comment:
It panics if there is no `VaultInfo` in the plan:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1439ce1]

goroutine 1 [running]:
github.com/elastic/cloud-on-k8s/hack/deployer/runner.(*GkeDriver).Execute(0xc000176dc0, 0xb, 0xb)
	/Users/michael/go/src/github.com/elastic/cloud-on-k8s/hack/deployer/runner/gke.go:82 +0x41
```

Reply:
555563b addresses that.